apps/prairielearn/src/lib/config.js

```diff
@@ -57,6 +57,9 @@ const ConfigSchema = z.object({
   authUin: z.string().nullable().default('000000000'),
   authnCookieMaxAgeMilliseconds: z.number().default(30 * 24 * 60 * 60 * 1000),
   sessionStoreExpireSeconds: z.number().default(86400),
+  sessionCookieSameSite: z
+    .string()
+    .default(process.env.NODE_ENV === 'development' ? 'lax' : 'none'),
```

Comment:
For anyone landing on this PR via `git blame`, we need `SameSite: None` for iframing in LTI tools like canvas. However, this breaks for local dev workflows that typically don't use TLS, as browsers will reject cookies with `SameSite: None` that don't also set `Secure`. This is our compromise: reasonable defaults with the ability to turn on `SameSite: None` locally if someone has TLS set up and wants to test iframes.